core/src/com/unciv/models/ruleset/unique/UniqueType.kt

```diff
@@ -571,12 +571,13 @@ enum class UniqueType(val text:String, vararg targets: UniqueTarget, val flags:
     ///////////////////////////////////////////// META /////////////////////////////////////////////
     
     
-    HiddenWithoutReligion("Hidden when religion is disabled", UniqueTarget.Unit, UniqueTarget.Building, UniqueTarget.Ruins),
-    HiddenBeforePantheon("Hidden before founding a Pantheon", UniqueTarget.Ruins),
-    HiddenAfterPantheon("Hidden after founding a Pantheon", UniqueTarget.Ruins),
-    HiddenAfterGreatProphet("Hidden after generating a Great Prophet", UniqueTarget.Ruins),
-    AvailableAfterCertainTurns("Only available after [amount] turns", UniqueTarget.Ruins),
-    HiddenWithoutVictoryType("Hidden when [victoryType] Victory is disabled", UniqueTarget.Building, UniqueTarget.Unit),
+    HiddenWithoutReligion("Hidden when religion is disabled", UniqueTarget.Unit, UniqueTarget.Building, UniqueTarget.Ruins, flags = listOf(UniqueFlag.HiddenToUsers)),
+    HiddenBeforePantheon("Hidden before founding a Pantheon", UniqueTarget.Ruins, flags = listOf(UniqueFlag.HiddenToUsers)),
+    HiddenAfterPantheon("Hidden after founding a Pantheon", UniqueTarget.Ruins, flags = listOf(UniqueFlag.HiddenToUsers)),
+    HiddenAfterGreatProphet("Hidden after generating a Great Prophet", UniqueTarget.Ruins, flags = listOf(UniqueFlag.HiddenToUsers)),
+    AvailableAfterCertainTurns("Only available after [amount] turns", UniqueTarget.Ruins, flags = listOf(UniqueFlag.HiddenToUsers)),
```

Comment:
This actually sounds like something the users should be aware of, no?

Reply:
Right.. Oops. Overzealous copy-pasting…